Project organization/Project structure.md

```diff
@@ -33,13 +33,13 @@ This is where most of the configuration for the application is located. We use i
 
 It is a good practice to group specific configuration sections like services and repositories to keep things organized. One of the ways to do this is through the implementation of IServiceBuilder extensions.
 
-Since we use [Options pattern](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1) for app configuration the bindings are done in the startup class. There are more details for this pattern in the Design pattern section of this handbook.
+Since we use the [Options pattern](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-3.1) for app configuration the bindings are done in the startup class. There are more details for this pattern in the Design pattern section of this handbook.
```

Comment:
This first sentence could use a comma: `Since we use the Options pattern for app configuration *,* the bindings are done in the startup class.`

Also, the last sentence is not actually true, we don't have any details about the Options pattern in our Design patterns section, but we do in the Configuration section. Please correct this sentence and add a link to that section.